test/Test/Oscoin/Configuration.hs

```diff
@@ -43,3 +59,16 @@ prop_roundtripNetworkShowRead :: Property
 prop_roundtripNetworkShowRead = property $ do
     net <- forAll Gen.enumBounded
     tripping net (show @Network) readEither
+
+prop_roundtripPathsCLI :: Property
+prop_roundtripPathsCLI = property $ do
```

Comment:
It might be nice a short description (in prose, perhaps with some inlined code) of what this test is supposed to return, even just to have a "wow factor" for this _tour de force_ 😉